packages/fxa-settings/src/components/ButtonDownloadRecoveryKey/index.tsx

```diff
@@ -0,0 +1,127 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { useAccount, useFtlMsgResolver } from '../../models';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { logViewEvent } from '../../lib/metrics';
+import { truncate } from 'fs';
```

Comment:
[`fs.truncate`](https://nodejs.org/api/fs.html#fstruncatepath-len-callback)? I don't think I'm familiar with that API. What's this for? (or, is it an accidental import since I'm not sure I see a `truncate()` usage below)

Reply:
Accidental! Now removed.